lib/index.js

```diff
@@ -59,9 +59,12 @@ bedrock.events.on('bedrock.init', async () => {
         referenceId: 'keyAgreementKey',
         required: true
       }, {
-        referenceId: 'assertionMethod:ed25519',
-        // FIXME: when other assertion methods are supported, make this `false`
-        required: true
+        referenceId: 'assertionMethod:Ed25519',
```

Comment:
We may also need a change somewhere to support issuer instances that have already been created with a zcap that uses the lower-cased `referenceId` -- probably where the issuer information is retrieved prior to signing. Or did you determine we don't need that?

It would probably be good to have a test for that ... which should be written without having to change the validation rules (so some database hack to change the zcap `referenceId` or a low-level call to create a config without going through the HTTP API for it or something?).

Reply:
Noted upper case formatting of keyType as breaking [316bd49](https://github.com/digitalbazaar/bedrock-vc-issuer/pull/113/commits/316bd495e4da4a7100dc948c8a9a6b745592ecf5)

Added check for backwards compatibility - [Add backwards compatibility check in getSuiteParams().](https://github.com/digitalbazaar/bedrock-vc-issuer/pull/113/commits/a40d46d6bfba1ee8a7b653a6fd9383350b002c91) & [daa5484](https://github.com/digitalbazaar/bedrock-vc-issuer/pull/113/commits/daa5484b93d225f351e6ac63970e8d7469730ab3)

Added backwards compatibility test - [6c28013](https://github.com/digitalbazaar/bedrock-vc-issuer/pull/113/commits/6c28013ca579c5033766fd622c846922eea0e006)